### PR TITLE
nsjail: update to 3.0

### PR DESCRIPTION
**Pull request by LamaRaz**
update nsjail to 3.0

### CODE REVIEW

**Comment by ericonr** [comment]
Your commit message should have the same content as the PR title.

File: srcpkgs/nsjail/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'nsjail'
 pkgname=nsjail
-version=2.9
+version=3.0
 revision=4
```
Comment (ericonr) [inline srcpkgs/nsjail/template:4]: Revision should go down to 1 after an update.

File: srcpkgs/nsjail/template [hunk 2]
```diff
@@ -11,7 +11,7 @@ license="Apache-2.0"
 homepage="http://nsjail.com"
 distfiles="https://github.com/google/nsjail/archive/${version}.tar.gz
 https://github.com/google/kafel/archive/722b93a630e53ca472a4c69f61cb3dfc44ea5041.tar.gz"
-checksum="f8578a48330d3d089c9234a2b562be08043f8fe77453c322724acdf403d0afc3
+checksum="cfa66d3ed136b2e221752287b95e544915e8a6760aa866f023b604d14a374919
  49d2327c794c90aca50a9933c00d453705534b16e5f42d8c8e19c3d274f5e5f8"
```
Comment (ericonr) [inline srcpkgs/nsjail/template:15]: The commit for kafel was changed as well, you should update it: https://github.com/google/nsjail/tree/3.0

**Comment by LamaRaz** [comment]
> Your commit message should have the same content as the PR title.

There doesn't appear to be the option to rename a commit.

**Comment by pullmoll** [comment]
@LamaRaz `git commit --amend` and later `git push -f`.

**Comment by ericonr** [comment]
You need to delete the patch in `srcpkgs/nsjail/patches` too.

**Comment by LamaRaz** [comment]
I'm currently using the github interface, if the naming of the commit is mandatory close this and I will refile.

**Comment by fosslinux** [comment]
Yes the naming of the commit is mandatory. I greatly suggest not using the github interface. It is impossible to test anything and rebase is impossible AFAIK. There is no need to close the PR, just checkout the branch on your local machine and rebase+push.

**Comment by fosslinux** [comment]
> There is no need to close the PR, just checkout the branch on your local machine and rebase+push.

.......

**Pull request closed without merging**